tsconfig.json

```diff
@@ -1,18 +1,18 @@
 {
   "compileOnSave": true,
   "compilerOptions": {
-    "target": "es5",
+    "experimentalDecorators": true,
+    "lib": ["es2015", "dom"],
     "noImplicitReturns": true,
```

Comment:
I think noImplicitReturns and noImplicitAny can be changed also

Reply:
@yjaeseok Oh, I will fix it. ASAP